data/splitContent.json

```diff
@@ -0,0 +1,27 @@
+[
+  {
+    "type": "splitContent",
+    "props": {
+      "children": [
+        [
```

Comment:
We do need some way to mark those split areas as different somehow, e.g. `do`, `dont`.